DataProcessing/DataConverter.cs

```diff
@@ -130,17 +132,31 @@ private void Process(DateTime date, string line)
                 {
                     _symbolIndex = Array.IndexOf(csv, "SYM");
                     _availableIndex = Array.IndexOf(csv, "AVAILABLE");
+                    _rebateRateIndex = Array.IndexOf(csv, "REBATERATE");
+                    _feeRateIndex = Array.IndexOf(csv, "FEERATE");
                 }
                 return;
             }
 
             var ticker = csv[_symbolIndex].Replace(' ', '.');
             var borrowableShares = csv[_availableIndex];
+            var rebateRateIndex = csv[_rebateRateIndex];
+            var feeRateIndex = csv[_feeRateIndex];
```

Comment:
We shouldn't assume `_rebateRateIndex/_feeRateIndex` will be there always, can happen for older dates, so here we should check if they are different than -1 before trying to access them